api/scpca_portal/models/project.py

```diff
@@ -55,6 +55,26 @@ def get_input_project_metadata_file_path():
     def computed_files(self):
         return self.project_computed_files.order_by("created_at")
 
+    @property
+    def ignored_additional_metadata_keys(self):
```

Comment:
two comments here:
* I don't think we need to have the individual mappings yet since we should be keeping everything in `additional_metadata` that isn't an attribute on the model itself
* I think we can remove the negation and just filter out the keys that we don't want to put in `additional_metadata` and just return the keys that we want to keep

Reply:
These ideas were great! Thank you for the suggestions, David!